src/components/ui/button.tsx

```diff
@@ -9,19 +9,18 @@ const buttonVariants = cva(
   {
     variants: {
       variant: {
-        default: "bg-interactive-accent text-on-accent shadow hover:bg-interactive-accent/90",
+        default:
+          "mod-cta bg-interactive-accent text-on-accent shadow hover:bg-interactive-accent-hover",
         destructive: "bg-modifier-error text-normal shadow-sm hover:bg-modifier-error/90",
-        outline:
```

Comment:
our button styles were confusing. Default looked identical to secondary, and outline doesn't render an outline button but more of a secondary button. I cleaned them up in this PR.